InnerEye/ML/model_training.py

```diff
@@ -152,6 +170,13 @@ def create_lightning_trainer(container: LightningContainer,
     callbacks = [best_checkpoint_callback, recovery_checkpoint_callback]
     if "callbacks" in kwargs:
         callbacks.append(kwargs.pop("callbacks"))  # type: ignore
+    is_azureml_run = not is_offline_run_context(RUN_CONTEXT)
+    progress_bar_refresh_rate = container.pl_progress_bar_refresh_rate
+    if progress_bar_refresh_rate is None and is_azureml_run:
+        # When running in AzureML, the default progress bar clutters the output files with thousands of lines.
+        progress_bar_refresh_rate = 50
```

Comment:
Does this need to be set here? I imagine most containers would override this if necessary (the FastMri container does), and this introduces a small hidden difference between an offline and AzureML run.